x-pack/platform/plugins/shared/streams_app/public/components/data_management/schema_editor/flyout/field_summary.tsx

```diff
@@ -66,6 +66,9 @@ export const FieldSummary = (props: FieldSummaryProps) => {
 
   const router = useStreamsAppRouter();
 
+  const parent = getParentId(stream.name);
```

Comment:
looks like the linked parent isn't necessarily the direct parent, it's `field.parent`. We should check whether that goes to the root